src/Appwrite/Utopia/Request.php

```diff
@@ -25,7 +25,11 @@ public function getParams(): array
         $parameters = parent::getParams();
 
         if (self::hasFilter() && self::hasRoute()) {
-            $endpointIdentifier = self::getRoute()->getLabel('sdk.namespace', 'unknown') . '.' . self::getRoute()->getLabel('sdk.method', 'unknown');
+            $method = self::getRoute()->getLabel('sdk.method', ['unknown']);
+            if (\is_array($method)) {
+                $method = $method[0];
+            }
```

Comment:
in what scenario is this an array

Reply:
https://github.com/appwrite/appwrite/blob/77b1762514125c319b2ae4d34f063812c86229d2/app/controllers/api/account.php#L198

We use array to define alias methods for the SDK to maintain backwards compatibility.